prepare.py

```diff
@@ -121,3 +121,6 @@ def prepare_netgen_plugin():
     prepare_geom()
     prepare_smesh()
     prepare_netgen_plugin()
+
+    pset = patch.fromfile('patch/noexcept.patch')
```

Comment:
Move function inside `prepare_smesh` function (likely at the end with a comment `# Patch noexcept` or something). Maybe after here https://github.com/trelau/SMESH/blob/ab0f4357a2fc8c250102bbf9ff2e057a9ca6a1aa/prepare.py#L98

Reply:
ok done. Is applying on src ok. Otherwise, I have to redo the replacement procedure.